### ISSUE TITLE
Transfer the task with the pimcore.events.postSaveObject Event in ExtJs

### ISSUE THREAD

**Issue by schmidt-max** [pimcore/pimcore#12796]
### Feature description

With the changed Event Handling (e.g. for Plugins) in ExtJs the property for the corresponding task (autosave, publish, ...) is missing. So it is not possible to handle the event concerning the caused task in the Event Handler (in our case we want to handle the event only if it is caused by the publish task and not for the autosave). So would it be possible to add this task again?

The object.js (pimcore/bundles/AdminBundle/Resources/public/js/pimcore/object/object.js)  with the added task:
```
  const postSaveObject = new CustomEvent(pimcore.events.postSaveObject, {
                                        detail: {
                                            object: this,
                                            task: task
                                        }
                                    });
```

**Comment by brusch** [comment]
I'd classify this as a bug, as it was there before, so it should be fixed in `10.5`.

**Comment by schmidt-max** [comment]
Perfect - thank you.
